browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -352,7 +364,10 @@ function _headerlistboxEntry(parentContainer, treeViewData, entry, builder) {
 	var disabled = treeViewData.enabled === 'false' || treeViewData.enabled === false;
 	var singleClick = _isSingleClickActivate(treeViewData);
 	var selectionElement;
-	if (entry.state !== undefined) {
+	if (entry.image) {
+		selectionElement = _createImageColumn(parentContainer, builder, entry);
```

Comment:
it should be created in the loop where we process columns - so we will put it in the correct place (sometimes on the start sometimes at the end or in the middle)